boards/arm/mercury_xu/Kconfig.defconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2019, Antmicro
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_MERCURY_XU
+
+config BOARD
+	default "mercury_xu"
+
+config BUILD_OUTPUT_STRIPPED
```

Comment:
Is this needed? I see this setting in 3 x86 boards, but I don't think it should be a default

Reply:
indeed, this is not needed. Removed